components/Statistics/Iceberg/Iceberg.vue

```diff
@@ -0,0 +1,33 @@
+<script lang="ts" setup>
```

Comment:
Wollen wir IceBerg auch in assets/vue verlagern?
Ich fände es übersichtlich, wenn alles in components/Statistics/ einen konkreten "Slide" im Karusell darstellt.